__tests__/app.test.js

```diff
@@ -22,3 +29,33 @@ describe("GET /api", () => {
       });
   });
 });
+
+
+describe("GET /api/topics", () => {
+    test("200: Responds with an array of topic objects, each object in the array should have properties: slug, description", () => {
+      return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics.length).toBe(3)
+        response.body.topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),    
+        })
```

Comment:
Great test! You could destructure the response like this:

`{ body: { topics } }`

Instead of writing response.body.topics, you grab topics directly from the response object 👍